ceph-releases/luminous/ubuntu/16.04/daemon/common_functions.sh

```diff
@@ -302,6 +302,19 @@ function get_part_uuid {
   blkid -o value -s PARTUUID "${1}"
 }
 
+#shellcheck disable=SC2120
+function is_dmcrypt () {
+  # As soon as we find partitions with TYPE=crypto_LUKS on ${OSD_DEVICE} we can
+  # assume this device is part of dmcrypt scenario.
+
+  # To keep compatibility with existing code
+  if [ -n "${1}" ]; then
+    local OSD_DEVICE
+    OSD_DEVICE="${1}"
+  fi
+  blkid -t TYPE=crypto_LUKS "${OSD_DEVICE}"* -o value -s PARTUUID &> /dev/null
```

Comment:
why `&> /dev/null`?

Reply:
we don't need the output, we just need the return code

Reply:
kk

Reply:
Then perhaps do `1>/dev/null` so `2` can be caught.

Reply:
updated